### PR TITLE
fixed #2123 trigger Event called native DOM event called twice

### LINKED ISSUE #2123: trigger Event called native DOM event called twice
Native DOM event with on prefix can be called twice when the trigger function is being called.

https://jsfiddle.net/h131rmck/

### PR DESCRIPTION
**Pull request by andycall**
trigger Event called native DOM event called twice

### CODE REVIEW

File: src/event.js [hunk 1]
```diff
@@ -316,7 +316,7 @@ jQuery.event = {
 			}
 
 			// Native handler
-			handle = ontype && cur[ ontype ];
+			handle = ontype && cur === elem &&  cur[ ontype ];
```
Comment (staabm) [inline src/event.js]: Nit: 1 space too much

**Comment by andycall** [comment]
I had signed CLA

**Comment by scottgonzalez** [comment]
You signed the CLA, but AndyCall isn't considered a valid name. Can you please sign with your full name?

**Comment by andycall** [comment]
sure, 
This problem is if I bind an addEventListener("click") instead of onclick , this double trigger will be disappear

**Pull request closed without merging**